pennylane_lightning/core/src/simulators/lightning_kokkos/gates/tests/Test_StateVectorKokkos_Param.cpp

```diff
@@ -978,10 +992,11 @@ TEMPLATE_TEST_CASE("StateVectorKokkosManaged::applySingleExcitationPlus",
         SECTION("Apply using dispatcher") {
             StateVectorKokkos<TestType> kokkos_sv{num_qubits};
             std::vector<ComplexT> result_sv(kokkos_sv.getLength(), {0, 0});
-
+            const TestType angle =
+                (inverse) ? -0.267030328057308 : 0.267030328057308;
             kokkos_sv.HostToDevice(ini_st.data(), ini_st.size());
-            kokkos_sv.applyOperation("SingleExcitationPlus", {0, 2}, false,
-                                     {0.267030328057308});
+            kokkos_sv.applyOperation("SingleExcitationPlus", {0, 2}, inverse,
+                                     {angle});
```

Comment:
If we negate the value and also pass the inverse value, should these cancel one another? Is it better to use one or the other?

Reply:
The goal here is to test both `inverse` implementations of the functor. I could have two set of reference values, but here I just flipped the angle instead.